node/helpers_test.go

```diff
@@ -0,0 +1,65 @@
+package node
+
+import (
+	"context"
+	"crypto/rand"
+	"errors"
+	"testing"
+	"time"
+
+	testutils "github.com/celestiaorg/utils/test"
+	"github.com/libp2p/go-libp2p/core/crypto"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/tendermint/libs/log"
+
+	mockda "github.com/rollkit/rollkit/da/mock"
+	"github.com/rollkit/rollkit/store"
+)
+
+func TestMockTester(t *testing.T) {
+	m := MockTester{t}
+	m.Fail()
+	m.FailNow()
+	m.Logf("hello")
+	m.Errorf("goodbye")
+}
+
+func TestGetNodeHeight(t *testing.T) {
+	require := require.New(t)
+	ctx := context.Background()
+	dalc := &mockda.DataAvailabilityLayerClient{}
+	ds, _ := store.NewDefaultInMemoryKVStore()
+	_ = dalc.Init([8]byte{}, nil, ds, log.TestingLogger())
+	_ = dalc.Start()
+	num := 2
+	keys := make([]crypto.PrivKey, num)
+	for i := 0; i < num; i++ {
+		keys[i], _, _ = crypto.GenerateEd25519Key(rand.Reader)
+	}
+	fullNode, _ := createNode(ctx, 0, false, true, false, keys, t)
+	lightNode, _ := createNode(ctx, 1, false, true, true, keys, t)
+	fullNode.(*FullNode).dalc = dalc
+	fullNode.(*FullNode).blockManager.SetDALC(dalc)
```

Comment:
we should create a follow up issue for some more helpers. feels like we should be able to just call one method for each node. Like `fullNode, _ := createFullNode()` and that handles all the `dalc` and `keys` generation when you don't need access to them like in this test.

Reply:
https://github.com/rollkit/rollkit/issues/985